app/controllers/openid_connect/logout_controller.rb

```diff
@@ -28,6 +48,21 @@ def index
       end
     end
 
+    def handle_successful_logout_request(redirect_uri)
+      if logout_params[:client_id] && logout_params[:id_token_hint].nil? && current_user
+        @client_id = logout_params[:client_id]
+        @state = logout_params[:state]
+        @post_logout_redirect_uri = logout_params[:post_logout_redirect_uri]
+        render :index
```

Comment:
Just checking: This line isn't strictly necessary, right? Since it's only ever called during the `index` action and would render this view by default anyways.

Reply:
It's not necessary, but I find the implicit render can be unintuitive, particularly when it's within conditionals